docs/master/api-reference/directives.md

```diff
@@ -1111,6 +1111,47 @@ class PlacedOrder
 }
 ```
 
+## @extends
+
+```graphql
+"""
+This allows types to extends other Types
+
+it inherits properties from the parent type. 
+It overides same attributes present in the parent type.
```

Comment:
This is not totally clear to me. Which attributes have precedence?

Reply:
That of the child.

Reply:
The description should be changed to make this completely disambiguous.